source/assets/js/helpers/module.js

```diff
@@ -12,7 +12,7 @@ class EstaticoModule {
 	* @param  {object} options - The options passed as data attribute in the Module
 	*/
 	constructor($element, _defaultData, _defaultOptions, data, options) {
-		this.name = this.constructor.name.toLowerCase();
+		this.name = this.constructor.name;
```

Comment:
@swey, so you would use `[data-' + this.name.toLowerCase() + '="bla"']` and `$domElement.data(this.name.toLowerCase())` instead?